jest.config.js

```diff
@@ -0,0 +1,20 @@
+const nextJest = require('next/jest')
+
+const createJestConfig = nextJest({
+  dir: './',
+});
+
+// Add any custom config to be passed to Jest
```

Comment:
This is fine to leave in but I'm wondering what this communicates. The variable name is `customJestConfig` which is pretty self-explanitatory. I'm not sure `// Add any custom config to be passed to Jest` adds additional clarity. Instead, documentation about what configuration settings are unique here _might_ be more important to include.

Not something you have to change for this PR, but something to consider for the future.

Reply:
I don't mind getting rid of the comment. I'm not particularly sure any of these configuration settings are unique right now.

Reply:
No comment: [7ecc487](https://github.com/sparkbox/accessible-components/pull/17/commits/7ecc487cd00990234ba01299165ba22ff28bb622)